### PR TITLE
Add common project repo details to README.md

### PR DESCRIPTION
**Pull request by estesp**
Signed-off-by: Phil Estes <estesp@linux.vnet.ibm.com>

### CODE REVIEW

**Comment by crosbymichael** [comment]
LGTM

File: README.md [hunk 1]
```diff
@@ -1,2 +1,56 @@
-# project
-Cross-project utilities, scripts, etc.
+# containerd common project repository
+This cross-project repository holds utilities, scripts, and common files
+used across the containerd master project and many sub-projects within
+the containerd organization.
+
+## containerd-release program
+
+The `containerd-release` Go program is maintained here in this project
```
Comment (dmcgowan) [inline README.md]: it got renamed to `release-tool` in this project

**Comment by estesp** [comment]
ah, hadn't looked closely enough; updated with correct name of tool @dmcgowan

**Comment by AkihiroSuda** [comment]
LGTM, but I noticed I'm not a maintainer of this repo...  :P  (I can't click the merge button)

**Comment by estesp** [comment]
ha.. I'll merge, but @crosbymichael can probably fix that

**Pull request merged**